grpc/src/main/java/com/linecorp/armeria/internal/grpc/ArmeriaMessageDeframer.java

```diff
@@ -267,6 +267,11 @@ public boolean isClosed() {
         return unprocessed == null;
     }
 
+    public ArmeriaMessageDeframer decompressor(Decompressor decompressor) {
+        this.decompressor = decompressor;
```

Comment:
I added a check for deframing already started as that'd be a big deal. Calling the method multiple times seems relatively harmless, so I thought it isn't worth adding new state just for that.